### PR TITLE
Docs of new Node.js SDK features

### PR DESCRIPTION
**Pull request by adams85**
### Description

Docs of new Node.js SDK features.

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/docs/sdk-reference/node.md [hunk 1]
```diff
@@ -82,54 +87,91 @@ _ConfigCat Client_ is responsible for:
...
+or
 
-Returns a Promise with the value.
+```js
+configCatClient
+  .getValueAsync(
+    'keyOfMySetting', // Setting Key
+    false, // Default value
+    { identifier: '435170f4-8a8b-4b67-a723-505ac7cdea92' },
+  ) // Optional User Object
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/node.md:157]: this comment is in the wrong line, does the formatter allow it in the above line?

Comment (adams85) [reply website/docs/sdk-reference/node.md]: Sharp eyes! 👍 Fixed by https://github.com/configcat/docs/pull/195/commits/697074f1f6153e1a6b3ef36caee4338cace795cb

On top of that, I also revised `User` object creation as the original code examples weren't completely correct. (Although `const user: User = { ... }` works, the created object won't have the correct prototype set which could lead to hard to track issues. Not in this case but I think it's better to advertise the correct usage.)

Comment (sigewuzhere) [reply website/docs/sdk-reference/node.md]: Absolutely yes! Thanks for bringing it up.

**Review by sigewuzhere — APPROVED** [review]

**Review by laliconfigcat — DISMISSED** [review]

File: website/docs/sdk-reference/node.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/node.md:1]:
```suggestion
```

**Review by laliconfigcat — CHANGES_REQUESTED** [review]
image
This new line creates some hell in the documentation.

**Review by laliconfigcat — APPROVED** [review]

**Pull request merged**